docs/quickstarts/express.mdx

```diff
@@ -107,21 +107,11 @@ Learn how to integrate Clerk into your Express backend for secure user authentic
   })
   ```
 
-  You can then use the [`getAuth`](/docs/references/express/overview#get-auth) helper to access the authentication state in any route:
```

Comment:
Removing this section as its redundant - its used in the next example and its explained in the overview. I don't think its necessary to add another code example like this